Compile error with 2nd_hw_i2c

I am trying to compile a script for the Atmega328PB using PlatformIO, where I got two OLED displays on HW I2C 1.
Currently, HW I2C 2 is not used.

The code (only excerpt of display declaration):
```cpp
#include <Arduino.h>
#include <Wire.h>
#include <U8g2lib.h>

U8G2_SSD1306_128X64_NONAME_1_HW_I2C display1(U8G2_R0, /* reset=*/ U8X8_PIN_NONE);
U8G2_SSD1306_128X64_NONAME_1_HW_I2C display2(U8G2_R0, /* reset=*/ U8X8_PIN_NONE);
```

but when I try to compile, I get this error:

```
.pio\libdeps\Upload_UART\U8g2\src\U8x8lib.cpp: In function 'uint8_t u8x8_byte_arduino_2nd_hw_i2c(u8x8_t*, uint8_t, uint8_t, void*)':
.pio\libdeps\Upload_UART\U8g2\src\U8x8lib.cpp:1368:7: error: 'Wire1' was not declared in this scope
```

A google search has led to nothing about this issue.
So how can I tell U8g2 not to worry about the second HW I2C bus?

--- a/cppsrc/U8x8lib.cpp
+++ b/cppsrc/U8x8lib.cpp
@@ -50,6 +50,11 @@
 #    include <i2c_t3.h>
 #  else
 #    include <Wire.h>
+#    ifdef U8X8_HAVE_2ND_HW_I2C
+#      if defined(MINICORE) && defined(__AVR_ATmega328PB__)
+#        include <Wire1.h>
+#      endif
+#    endif
 #  endif
 #endif /* U8X8_HAVE_HW_I2C */
 

